worker/provision_device.json

```diff
@@ -1,29 +1,46 @@
 {
-    "name": "provision_device",
-    "description": "Provision device.",
-    "version": 2,
-    "tasks": [
-        {
-            "name": "create_device_inventory",
-            "type": "http",
-            "retries": 3,
-            "http": {
-                "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
-                "method": "POST",
-                "contentType": "application/json",
-                "body": "${workflow.input.device}",
-                "headers": {
-                    "X-MEN-RequestID": "${workflow.input.request_id}",
-                    "Authorization": "${workflow.input.authorization}"
-                },
-                "connectionTimeOut": 8000,
-                "readTimeOut": 8000
-            }
-        }
-    ],
-    "inputParameters": [
-        "request_id",
-        "authorization",
-        "device"
-    ]
+  "name": "provision_device",
+  "description": "Provision device.",
+  "version": 3,
+  "tasks": [
+    {
+      "name": "create_device_inventory",
+      "type": "http",
+      "retries": 3,
+      "http": {
+        "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
+        "method": "POST",
+        "contentType": "application/json",
+        "body": "{\"id\":\"${workflow.input.device_id}\"}",
+        "headers": {
+          "X-MEN-RequestID": "${workflow.input.request_id}",
+          "Authorization": "${workflow.input.authorization}"
+        },
+        "connectionTimeOut": 8000,
+        "readTimeOut": 8000
+      }
+    },
+    {
+      "name": "provision_deviceconnect",
```

Comment:
I'm not 100% sure, but I believe device connect is going to be an optional component. Is it right?
If this is the case, we should add statusCodes to avoid failure if the internal API end-point is not available.
See: https://github.com/mendersoftware/workflows-enterprise/pull/109/files#diff-ca7b3e6a2739d529e9bbfc6f1a28a87612206302053d265ee1b4eb84a835dd18R85

Reply:
Done!